### PR TITLE
limit: use a dummy var if neither symvar nor user give one

### LINKED ISSUE #669: limit uses symvar but doesn't check for empty return
Currently it seems to work but I don't see that its supported from reading the SymPy `limit` documentation: in particular, it only works because `limit` has `Limit(e, z, z0, dir).doit(deep=False)`.  If you change that to `deep=True` then it fails if `z` is empty.

I think our limit should just use some dummy variable like `x` if `symvar` returns empty.

### PR DESCRIPTION
**Pull request by cbm755**
Fixes #669.

### CODE REVIEW

File: inst/@sym/limit.m [hunk 1]
```diff
@@ -83,6 +83,10 @@
       print_usage ();
   end
 
+  if (isempty (x))
+    x = sym('x');
```
Comment (latot) [inline inst/@sym/limit.m]: if x is empty, can we just return f?, without a limit var, the limit will be the same function.

Comment (cbm755) [inline inst/@sym/limit.m]: That makes sense, done.

**Comment by cbm755** [comment]
I fixed another small bug I found.  Should have done a separate PR to practice what I preach but here we are.

**Comment by cbm755** [comment]
Will merge assuming tests pass.

**Pull request merged**